deps/rabbit/src/rabbit.erl

```diff
@@ -175,6 +175,12 @@
                     {requires,    [rabbit_alarm, guid_generator]},
                     {enables,     core_initialized}]}).
 
+-rabbit_boot_step({rabbit_queue_member_eval,
```

Comment:
Could we no just have the process as static configuration?

Reply:
I am all for that, just tried to 'follow suit' from other supervised processes and creation of supervisors, so piggybacked on that code.